dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -776,7 +776,7 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
     @ApiOperation(value = "Get the authenticated user's starred tools.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Entry.class, responseContainer = "List")
     public Set<Entry> getStarredTools(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user) {
         User u = userDAO.findById(user.getId());
-        return u.getStarredEntries().stream().filter(element -> element instanceof Tool)
+        return u.getStarredEntries().stream().filter(element -> element instanceof Tool || element instanceof AppTool)
```

Comment:
Some day, not for this PR, this could could be a JPA query. Although most/all users probably don't have enough starred entries where this would be significant.